lib/firebase.ts

```diff
@@ -0,0 +1,16 @@
+// @ts-nocheck
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+
+const firebaseConfig = {
+  apiKey: FIREBASE_API_KEY,
```

Comment:
The firebase config can go directly here

Reply:
but wouldn't it be unprotected by placing it directly here?

Reply:
It's fine this info can be public